### PR TITLE
Created problem_37 in project_euler

### PR DESCRIPTION
**Pull request by Kush1101**
### **Describe your change:**

* [ ] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [ ] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by Kush1101** [comment]
@cclauss. Time for a review.

File: project_euler/problem_37/sol1.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        if validate(num):
+            list_nums = list_truncated_nums(num)
+            if all(is_prime(i) for i in list_nums):
+                list_truncated_primes.append(num)
+        num += 2
+    return list_truncated_primes
+
+
+if __name__ == "__main__":
+    print(f"{sum(compute_truncated_primes(11))}")
```
Comment (cclauss) [inline project_euler/problem_37/sol1.py:92]:
```suggestion
    print(f"{sum(compute_truncated_primes(11)) = }")

```

Comment (Kush1101) [reply project_euler/problem_37/sol1.py:92]: Isn't this invalid?

Comment (Kush1101) [reply project_euler/problem_37/sol1.py:92]: If I run this I get a syntax error message

Comment (cclauss) [reply project_euler/problem_37/sol1.py:92]: Python -V == Python 3.8?!?

Comment (Kush1101) [reply project_euler/problem_37/sol1.py:92]: I am on 3.7

Comment (Kush1101) [reply project_euler/problem_37/sol1.py:92]: Shall I commit it?

Comment (cclauss) [reply project_euler/problem_37/sol1.py:92]: Yes.  This was a change made in 3.8

**Comment by Kush1101** [comment]
Thanks!

**Pull request merged**